sunpy/visualization/__init__.py

```diff
@@ -1,2 +1,3 @@
+from sunpy.visualization.drawing import *
```

Comment:
I think we don't want to import everything in `drawing` into the `visualization` namesapce?
```suggestion
```